packages/imperative/src/doc/IImperativeEnvironmentalVariableSettings.ts

```diff
@@ -47,4 +47,10 @@ export interface IImperativeEnvironmentalVariableSettings {
      * @type {IImperativeEnvironmentalVariableSetting}
      */
     maskOutput?: IImperativeEnvironmentalVariableSetting;
+
+    /**
+     * The directory where CLI plugins are installed.
+     * Default is `${cliHome}/plugins`.
+     */
+    pluginsDir?: IImperativeEnvironmentalVariableSetting;
```

Comment:
Thoughts on keep using annotations like `@type` ?

It doesn't affect the generated `typedoc`, Examples:
- with types defined: [IImperativeEnvironmentalVariableSettings](https://docs.zowe.org/stable/typedoc/interfaces/_zowe_imperative.iimperativeenvironmentalvariablesettings.html)
- without types defined: [IImperativeAuthGroupConfig](https://docs.zowe.org/stable/typedoc/interfaces/_zowe_imperative.iimperativeauthgroupconfig.html)

It appears in VSCode suggestions
![image](https://user-images.githubusercontent.com/37381190/186924711-3f9c4abc-71c2-48de-a219-8c5ddad732d0.png)

However, the type already says it's a string, and you already know the parent class/interface by hovering over the object, so `@memberOf` could also be removed.
![image](https://user-images.githubusercontent.com/37381190/186925366-ec4467bb-06a2-4104-b0aa-9e8926099ec8.png)

Reply:
I have never been a fan of our (often redundant) typedoc annotations. I would not suggest that we make a concentrated effort to remove them everywhere. However, if we choose to not place them in new source files, and remove them when convenient in a file that we are already working on, we could remove clutter that distracts someone who is reading the code.